packages/core/src/__integrationtests__/Ctypes.spec.ts

```diff
@@ -23,8 +23,17 @@ import { disconnect } from '../kilt'
 import { createEndowedTestAccount, initializeApi, submitTx } from './utils'
 
 let api: ApiPromise
+// we skip fetching CTypes from chain for the old pallet version, where the necessary information was not yet on chain.
+let hasBlockNumbers: boolean
 beforeAll(async () => {
   api = await initializeApi()
+  // this extrinsic should only exist in the new pallet version
+  hasBlockNumbers = typeof api.tx.ctype.setBlockNumber !== 'undefined'
+  if (!hasBlockNumbers) {
+    console.warn(
```

Comment:
This `console.warn` triggers a lint warning. Is there a workaround? Maybe simply skipping it?

Reply:
or a linter exception :D

Reply:
Yeah I meant skipping the warning with a linter exception 😁

Reply:
actually that's something we can disable for all tests I think (https://github.com/KILTprotocol/sdk-js/pull/703/commits/ba18a01b1c480533470839794d5e6e53f0c7a000)